ballerina/tests/bytes_io.bal

```diff
@@ -142,38 +96,31 @@ isolated function testFileWriteBytesFromStreamUsingIntermediateFile() returns Er
 
     check fileWriteString(resourceFilePath, content);
     stream<Block, Error?> bytesStream = check fileReadBlocksAsStream(resourceFilePath, 2);
-    var result = fileWriteBlocksFromStream(filePath, bytesStream);
-    if (result is Error) {
-        test:assertFail(msg = result.message());
-    }
-    return;
+    check fileWriteBlocksFromStream(filePath, bytesStream);
 }
 
 @test:Config {dependsOn: [testFileWriteBytesFromStreamUsingIntermediateFile]}
 function testFileReadBytesAsStreamUsingIntermediateFile() returns Error? {
     string filePath = TEMP_DIR + "bytesFile3_A.txt";
-    var result = fileReadBlocksAsStream(filePath, 2);
+    stream<Block, Error?> result = check fileReadBlocksAsStream(filePath, 2);
     string expectedString = "Sheldon Cooper";
     byte[] byteArr = [];
-    if (result is stream<Block, Error?>) {
-        check result.forEach(function(Block val) {
-                                      foreach byte b in val {
-                                          byteArr.push(b);
-                                      }
-                                  });
-        string|error returnedString = langstring:fromBytes(byteArr);
-        if (returnedString is string) {
-            test:assertEquals(returnedString, expectedString);
-        } else {
-            test:assertFail(msg = returnedString.message());
+    check result.forEach(function(Block val) {
+        foreach byte b in val {
+            byteArr.push(b);
         }
+    });
+    string|error returnedString = langstring:fromBytes(byteArr);
```

Comment:
```suggestion
    string returnedString = check langstring:fromBytes(byteArr);
    test:assertEquals(returnedString, expectedString);
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-io/pull/420/commits/917a8e482c0c914bc2ca9970fe31f935023fbdf4